docs/sdk-json-proposal/SDK_API_WORKFLOW.md

```diff
@@ -0,0 +1,132 @@
+# Mesh Data Plane SDK – API Workflow
+## General Workflow
+1. Create a Mesh client:
+   * `mesh_create_client()`
+1. Create a Mesh Tx or Rx connection:
+   * `mesh_create_tx_connection()`
+   * `mesh_create_rx_connection()`
+1. Get a buffer from the Mesh connection:
+   * `mesh_get_buffer()`
+   * `mesh_get_buffer_timeout()`
+1. Put the buffer to the Mesh connection:
+   * `mesh_put_buffer()`
+   * `mesh_put_buffer_timeout()`
+1. Shutdown the Mesh connection:
+   * `mesh_shutdown_connection()`
+1. Delete the Mesh connection:
+   * `mesh_delete_connection()`
+1. Delete the Mesh client:
+   * `mesh_delete_client()`
+
+## Configure Connection – Multipoint Group
+```json
+"connection": {
+  "multipoint-group": {
+    "ipAddr": "224.0.0.1",
+    "port": "9003"
+  }
+}
+```
+
+## Configure Connection – SMPTE ST2110-XX
+```json
+"connection": {
+  "st2110": {
+    "transport": "st2110-22",
+    "remoteIpAddr": "192.168.95.2",
+    "remotePort": "9002",
+    "pacing": "narrow",
+    "payloadType": 112
+  }
+}
+```
+
+## Configure Connection – RDMA
+```json
+"connection": {
+  "rdma": {
+    "connectionMode": "RC",
+    "maxLatencyNs": 10000
+  }
+}
+```
+
+## Configure Payload – Video
+```json
+"payload": {
+  "video": {
+    "width": 1920,
+    "height": 1080,
+    "fps": 60.0,
+    "pixelFormat": "yuv422p10le"
+  }
+}
+```
+
+## Configure Payload – Audio
+```json
+"payload": {
+  "audio": {
+    "channels": 2,
+    "sampleRate": 48000,
+    "format": "pcm_s24be",
+    "packetTime": "1ms"
+  }
+}
+```
+
+## Configure payload – Blob data
+```json
+"payload": {
+  "blob": {
+   ...
+  }   
+},
+"maxPayloadSize": 2097152
```

Comment:
Remove ```"maxPayloadSize": 2097152``` from "payload - Blob data"

Please add additional point with additional configuration properties :
```
  "bufferQueueCapacity": 16,
  "maxPayloadSize": 2097152,
  "maxMetadataSize": 8192,
```

Reply:
I don't get it. Can you use the Suggestion mode here in Github PR?

Reply:
How about that ?

## Configure payload – Blob data
```json
"payload": {
  "blob": {
   ...
  }   
}
```
## Configure additional parameters
```json
"maxPayloadSize": 2097152,
"bufferQueueCapacity": 16,
```